getstarted.adoc

```diff
@@ -0,0 +1,92 @@
+---
+layout: getstarted
+title: Get Started with WildFly
+permalink: /get-started/
+---
+:page-liquid:
+
+== Build and run a Jakarta EE application with WildFly in a few minutes.
+
+=== Step 0. Install Java & Maven
+
+You need Java (at least version `11`, and preferably the latest LTS version) and Maven installed on your machine to create a Maven project that contains the source code of the Jakarta EE application.
```

Comment:
For now...

s/the latest LTS version/`17`/g

See https://www.wildfly.org/news/2023/10/18/WildFly30-Released/ and 'Recommended SE Version' for why.

In the future I want this to be driven by metadata in _data/releases.yml, but that's a separate task.